### DISCUSSION TITLE
Stripped-down build with only the drivers I need

### DISCUSSION THREAD

**Discussion by megatog615** [ipxe/ipxe#1352] [Q&A]
Been trying to build ipxe in the following way:
`make bin/rtl8139.rom`
Unfortunately it outputs a 70KiB rom file which is too big for my application. I notice in the build log that it's building things like "3c509.o" and "3c503.o" etc which are completely unnecessary drivers in my case. I have looked in the wiki and I cannot find how to simply build a rom with one network card driver.

Is there a way to do this or am I misinterpreting the build log?

**Comment by NiKiZe** [comment]
Ipxe builds all files, but it only links the driver you have requested.

To reduce build size, see named configs, such as vbox

**Reply by megatog615** [reply]
I had a feeling that's what was happening.

I tried the vbox config and the resulting rom is 63.5KiB which while smaller isn't enough of a savings to make it fit.
